src/shared/sym_load.c

```diff
@@ -46,6 +46,22 @@ void* so_get_module_handle(const char *so){
 #endif
 }
 
+void* so_check_module_loaded(const char *so){
+#ifdef WIN32
+    char file_name[MAX_PATH] = { 0 };
+    snprintf(file_name, MAX_PATH-1, "%s.dll", so);
+    return GetModuleHandle(file_name);
+#else
+    char file_name[4096] = { 0 };
```

Comment:
Can we define this value outside this conditional?

Reply:
I followed the same convention than the rest of the functions defined here.
The thing is that in Windows, `MAX_PATH` is 260 by default, not 4096
https://learn.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=registry